dali/util/npp.cc

```diff
@@ -35,4 +35,40 @@ int NPPInterpForDALIInterp(DALIInterpType type, NppiInterpolationMode *npp_type)
   return DALISuccess;
 }
 
+namespace {
+
+NppStreamContext GetNppContextImpl() {
+  NppStreamContext ctx;
+  ctx.hStream = nullptr;
```

Comment:
```suggestion
  ctx.hStream = 0;
```
It is a pointer internally but 0 is always referenced as a default one.